nucypher/policy/conditions/lingo.py

```diff
@@ -330,6 +330,18 @@ def __init__(self, operand: Condition):
     "float": lambda a: float(a),
     "int": lambda a: int(a),
     "str": lambda a: str(a),
+    # JSON conversion
+    "fromJson": lambda a: json.loads(a if isinstance(a, str) else a.decode()),
+    "toJson": lambda a: json.dumps(a),
+    # hex conversion
+    "fromHex": lambda a: bytes(HexBytes(a)),
+    "toHex": lambda a: (
+        HexBytes(a).hex()
+        if isinstance(a, (bytes, bytearray))
+        else HexBytes(a.encode() if isinstance(a, str) else str(a).encode()).hex()
+    ),
+    # hashing
+    "keccak": lambda a: HexBytes(keccak(a.encode() if isinstance(a, str) else a)).hex(),
```

Comment:
Why automatically convert the hash to hex? The user has the option to pair it with the use of "toHex" operation if they want hex.

```suggestion
    "keccak": lambda a: keccak(a.encode() if isinstance(a, str) else a),
```